examples/websites/README.md

```diff
@@ -0,0 +1,53 @@
+#ipfs for websites
+### A short guide to hosting your site on ipfs
+
+Adding your static website to ipfs is quite simple! Simply turn on your daemon:
+```bash
+$ ipfs daemon
+```
+
+And add the directory containing your website:
+```bash
+$ ls mysite
+index.html
+$ ipfs add -r mysite
+added QmbRyftGAtNjXs7hWZUTLyp27qrpgRUXKEwXCVPBueZGek mysite/index.html
+added QmYAMp9ptyn3Bv8ijoYoYAAQHFCfpZ1NKRNedkEWrR9DPH mysite/
```

Comment:
do these hashes work? we should move to making all hashes used in examples be real. (i'm bad at this, but we should do it)